config/locales/en.yml

```diff
@@ -77,10 +77,10 @@ en:
       region: *region
   deferral_details:
     view:
-      deferred_before_starting: Trainee deferred before their ITT started
+      deferred_before_starting: Trainee deferred before their <abbr title="Initial Teacher Training">ITT</abbr> started
```

Comment:
```suggestion
      deferred_before_starting: Trainee deferred before their <abbr title="initial teacher training">ITT</abbr> started
```

capitalisation rule is — lower case everything (unless it’s at the start of a sentence).